src/Utils/Logger.ts

```diff
@@ -16,6 +16,8 @@
 
 import * as vscode from 'vscode';
 
+type MsgList = (number|boolean|string|object)[];
```

Comment:
Is it meaning that `MsgList` is a list for number, boolean, string and object?

Reply:
If object is a type of the `MsgList`, it's meaning that the list is all of types.

Reply:
> Is it meaning that MsgList is a list for number, boolean, string and object?
> If object is a type of the MsgList, it's meaning that the list is all of types.

Yes. correct. :-)
I added more comment on the above samples: 
- https://github.com/Samsung/ONE-vscode/pull/710#discussion_r881416019 
- https://github.com/Samsung/ONE-vscode/pull/710#discussion_r881414291